portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncAttributes.testcase

```diff
@@ -828,4 +828,37 @@ definition {
 		}
 	}
 
+	@description = "Story: LRAC-10867 | Automation ID: LRAC-12911 | Test Summary: Check that it is possible to search the Attribute for Orders list"
+	@priority = "4"
+	test CanSearchAttributeInOrdersList {
+		var searchedOrdersAttribute = "orderId";
+		var unsearchedOrdersAttribute = "currencyCode,orderDate";
+
+		task ("Click to select attribute for orders") {
+			ACDXPSettings.clickToSelectAttributes(attributeCategory = "Account");
```

Comment:
Hey @calvinctvo, I think this should be **Order** here. Remember to run the tests to verify that it is running without errors.